JavaScript/Topics/13.DOM_in_javascript/README.md

```diff
@@ -0,0 +1,140 @@
+
+
+# JavaScript HTML DOM
+
+
+
+## What is DOM ?
+ DOM stands for Document object model . It is an application programming interface for manipulating HTML and XML documents. The DOM represents a document as a tree of nodes . It allows programs and scripts to dynamically access and update the content, structure, and style of a document.
+
+
+Below image shows a basic heirachy in DOM :
+
+
+![DOM ](https://searchengineland.com/figz/wp-content/seloads/2015/05/Microsoft-PowerPoint.jpg)
+
+
+## Use of DOM in javascript 
+<br>
+With the DOM , javascript get all the power it needs to create dynamic HTML . With DOM :
+
+* JavaScript can change all the HTML elements in the page
+
+* JavaScript can change all the HTML attributes in the page
+
+* JavaScript can change all the CSS styles in the page
+
+* JavaScript can remove existing HTML elements and attributes 
+
+* JavaScript can add new HTML elements and attributes
+
+* JavaScript can react to all existing HTML events in the page
+
+* JavaScript can create new HTML events in the page
+
+
+## Document object in JS
```

Comment:
```suggestion
## Document object in JavaScript
```
whatever you write JS or javascript, write them in a same format, these things doesn't matters at all, but for a clean doc it really matters.